code/modules/vending/vending.dm

```diff
@@ -1707,6 +1707,71 @@ ABSTRACT_TYPE(/obj/machinery/vending/cola)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/communications, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/teleporter, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/secure_data, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/atmospherealerts, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/atmospheresiphonswitch, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/air_management, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/air_management, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/injector_control, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/general_alert, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/atm, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/robot_module_rewriter, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/cloning, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/genetics, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/tetris, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/arcade, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/solar_control, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/powermonitor, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/powermonitor_smes, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/barcode, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/barcode_qm, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/operating, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/qmorder, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/qmsupply, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/stockexchange, amount=2, cost=PAY_UNTRAINED/3)
+		product_list += new/datum/data/vending_product(/obj/item/circuitboard/transception, amount=2, cost=PAY_UNTRAINED/3)
```

Comment:
Should have a check here for if its nadir so you dont end up with pointless transception controls